packages/react/src/Pagination/Pagination.tsx

```diff
@@ -91,17 +91,8 @@ const Page = toggleStyledComponent(
       box-shadow: inset 0 0 0 3px ${get('colors.fg.onEmphasis')};
     }
 
-    &[aria-disabled],
-    &[aria-disabled]:hover {
-      margin: 0 2px;
-
-      &:first-child {
-        margin-right: 6px;
-      }
-    }
-
-    &[aria-disabled],
-    &[aria-disabled]:hover,
+    &[aria-hidden],
+    &[aria-hidden]:hover,
```

Comment:
This will require duplicate changes in https://github.com/primer/react/blob/5a2568a0fae33457d566749fc4c855bc155a37ed/packages/react/src/Pagination/Pagination.module.css for now as we're migrating the sx styling out of prc